ui/admin/help-resources.php

```diff
@@ -1,21 +1,21 @@
-<h3>Pods Support</h3>
+<h3><?php _e('Pods Support', 'pods'); ?></h3>
 
-<p>There are many resources available to help you learn to use Pods on our site, <a href="http://pods.io/" target="_blank">Pods.io</a>:</p>
+<p><?php _e('There are many resources available to help you learn to use Pods on our site,', 'pods'); ?><a href="http://pods.io/" target="_blank">Pods.io</a>:</p>
 <ul class="ul-disc">
-	<li>To learn more about using Pods, see the <a href="http://pods.io/docs/" target="_blank">documentation</a>, or our <a href="http://pods.io/tutorials/" target="_blank">tutorials section</a>.
+	<li><?php _e('To learn more about using Pods, see the', 'pods'); ?><a href="http://pods.io/docs/" target="_blank">documentation</a>, or our <a href="http://pods.io/tutorials/" target="_blank">tutorials section</a>.</li>
 
-	<li>To get help with a specific issue, you can ask in our <a href="http://pods.io/forums/" target="_blank">support forums</a>, or in <a href="http://pods.io/forums/chat/" target="_blank">our chat</a>.
+	<li><?php _e('To get help with a specific issue, you can ask in our <a href="http://pods.io/forums/" target="_blank">support forums</a>, or in <a href="http://pods.io/forums/chat/" target="_blank">our chat</a>.', 'pods'); ?></li>
 
-	<li>To report <strong>bugs or request features</strong>, go to our <a href="https://github.com/pods-framework/pods/issues?sort=updated&direction=desc&state=open" target="_blank">GitHub</a>.</li>
+	<li><?php _e('To report <strong>bugs or request features</strong>, go to our <a href="https://github.com/pods-framework/pods/issues?sort=updated&direction=desc&state=open" target="_blank">GitHub</a>.', 'pods');  ?></li>
```

Comment:
@NikV Don't put HTML in the translation string. See: http://ottopress.com/2012/internationalization-youre-probably-doing-it-wrong/ Heading "Annoyance the First: Thou shalt not put unnecessary HTML markup into the translated string." 

I have 100% adopted Otto's opinions on this topic that are expressed in that article. Please assimilate said knowledge.